### ISSUE TITLE
Batch update items in inventory

### ISSUE THREAD

**Issue by git-wil** [HMC-Makerspace/MAKE#50]
# Goal
It can be useful to apply a batch update to many items in inventory, like modifying quantity, location, etc. for a large group of items. It would be convenient to have a method to batch-update items using a query and a mod to update all items that follow from that query.

Should be completed after #28.

# Subtasks
- [ ] Define a discrete list of modifications that can be applied
- [ ] Define a query language/system for querying items (may be just by text, may be more complex)
- [ ] Create a popup/page/kiosk for inputting queries and batch updates
 - [ ] This should show a preview of the modifications before actually applying them
 - [ ] Would be nice to have an undo/backup feature, but may be difficult to implement
